src/locales/fi.json

```diff
@@ -486,6 +486,7 @@
       "title": "ASIAKKAAN VENEET",
       "boatInfo": "VENEEN TIEDOT",
       "editBoat": "Muokkaa",
+      "createBoat": "Lisää uusi",
```

Comment:
it's a common button present in other sections as well, I would either put it in `"common"` or in `"customerView"`

Reply:
Will change both `editBoat` and `createBoat` to one from common.

Reply:
"Lisää uusi" did not actually exist in the translations before, only "Lisää" was under common. In any case, both "Muokkaa" and "Lisää uusi" are now resolved from common.